tf_files/shared/modules/k8s_configs/creds.tpl

```diff
@@ -21,7 +21,7 @@
         "data_delivery_bucket_aws_key_id": "${amanuensis-bot_user_id}",
         "data_delivery_bucket_aws_access_key": "${amanuensis-bot_user_secret}"
     },
-    "gearbox": {
+    "gearboxaws": {
```

Comment:
@paulmurdoch19 do we need to update the backend and any other service pulling this config to use gearboxes instead of gearbox or this is on a higher level of abstraction and the kube-setup command does that already?

Reply:
yes this portion of creds.json gets dumped into gearbox.env which is added to the gearbox container so the backend never sees this

Reply:
Do I have to delete the creds.json for it to create it with gearboxaws instead of gearbox? in portal dev it is already present and rolling nothing changed in that file. This I assume would break the creation of the gearbox.env file since that json attribute is not present. 

Delete I assume it could cause problems like not being able to connect to the DB anymore or create new DBs, but I am not sure about this as I have not checked recently that piece of code.

Reply:
Also for some reason there are more values under this voice than the ones listed in the template file. Not sure if those were added manually or programmatically. Could you compare with the version you started in the playground AWS account?
<img width="199" alt="image" src="https://github.com/user-attachments/assets/765d9b8e-5d6d-428c-ba96-98b55c991df9">

Reply:
1) you wont have to delete that file since it is created during the 1st phase of the terraform build. you would just need to edit the creds.json file and change the key "gearbox" to "gearboxaws" to avoid breaking the creation of the gearbox.env file. Alternatively you could tear down the whole thing but that is probably not worth it and will cause more problems. I am also not sure on this but I think if you delete the creds.json file and roll it it might try and spawn a new one.
2) and those db values in "gearbox" were added manually to avoid the error this PR fixes. Essentially how to DB creation works is that if you attempt to create a DB its first going to check to see if the db name exists inside creds.json so in this case gearbox does and then it works off the assumption that if the service name is in the creds.json then a server was created for that service as part of the terraform and therefore gen3 will not create a DB for that service. Thats why amanuensis, fence and sheepdog are in there but arborist is not.

Reply:
let double check all this though

Reply:
About pint 2 - should we add those keys in the cress template file tho, or they will be added automatically now that 'gearbox' is not present in that file?

Reply:
by cress template file, are you referring to the creds.tpl template file in cloud automation? If you are then no since the db server rotates every time you start up gen3, so when a gearbox DB is created the information is stored in a dbcreds.json file in Gen3secrets/g3auto/gearbox and that file is what gets pulled when creating the gearbox.env file